### PR TITLE
Fixes for errors.md and inline-queries.md

### PR DESCRIPTION
**Pull request by CalsiBotDev**

### CODE REVIEW

**Review by KnorpelSenf — APPROVED** [review]
LGTM. You could some painful typos like choose/chose etc—thanks!

**Comment by KnorpelSenf** [comment]
@linbuxiao @WingLim @taotie111 please update the Chinese translation for `errors.md` accordingly.

File: site/docs/guide/errors.md [hunk 1]
```diff
@@ -22,14 +22,14 @@ A more advanced error handling mechanism can be found [down here](#error-boundar
 
 ## Catching Errors
 
-It depends on your setup how to catch errors.
+How you catch errors will depend on your setup.
```
Comment (KnorpelSenf) [inline site/docs/guide/errors.md:25]: It should be only this sentence.

Comment (WingLim) [reply site/docs/guide/errors.md:25]: Updated in 40ba06f66a9748eb02a2256b01350a62eef0b18a

**Pull request merged**